juego.py

```diff
@@ -292,18 +292,18 @@ def load_image(name):
     return pygame.image.load(path).convert_alpha()
 
 # Funcion para cargar Sonidos
-def load_sound(name, activity_root):
-    path = os.path.join(activity_root,'data',name)
+def load_sound(name):
+    path = os.path.join('./data',name)
```

Comment:
I am not a fan of this. I would suggest, 
```python
os.path.join('.', 'data')
```
to ensure consistency. But I am suspicious whether it works when installed system-wide

Reply:
> But I am suspicious whether it works when installed system-wide

Does your suspicion point to removal of `get_activity_root()`?
I checked the path which is specified by using this function and it does not have the audio files- therefore the load error. Also [documentation](https://people.sugarlabs.org/sam/sugar3.activity.activity.html#sugar3.activity.activity.Activity.get_activity_root) suggests that it was used to `save` activity specific stuff. Please correct me if I wrongly interpreted the function use.

Reply:
No, not that. the `.` gets the current directory, I am not sure / I don't remember if `sugar-activity3` always `chdir` to the activity path before running. @quozl would know, I did run it, and it worked for me. But I do not understand the logic, possibly, `sugar-activity3` chdir's to the activity root before starting the activity.

Reply:
> quozl would know

I've since forgotten and would have to test.  Watch out for code that works when the activity is installed by download in `~/Activities` and doesn't work when installed by a package in `/usr/share/sugar/activities`.  The latter scenario is new, compared to the age of some of the code being worked on lately.

Reply:
I tested it in `/usr/share/sugar/activities`; It worked for me :confused: 

> sugar-activity3 chdir's to the activity root before starting the activity.

Probably not. Had that been the case, it wouldn't work when run from terminal with `sugar-activity3` using the changes done in 65a05e1

Reply:
> to ensure consistency. But I am suspicious whether it works when installed system-wide

It'll work because the activity root gets created the first time the activity is run and at the time no sounds were added to the data directory but the sounds exist in the data directory of the activity hence looking in the activity directory for the sounds make sense.

Reply:
> It'll work

Great! So is it safe to assume that this conversation is resolved?

Reply:
Yes, but, I prefer this 
```python
os.path.join('.', 'data')
```
or just 
```
'data'
```
because its always relative to the cwd